libraries/libfc/src/crypto/bls_private_key.cpp

```diff
@@ -49,6 +49,10 @@ namespace fc::crypto::blslib {
       return std::string(config::bls_private_key_prefix) + data_str;
    }
 
+   bool operator == ( const bls_private_key& pk1, const bls_private_key& pk2) {
+      return std::memcmp(&pk1, &pk2, sizeof(pk1)) == 0;

```

Comment:
I think the comment I added was important, because ideally we shouldn't do `memcmp`.
Also better to compare the member `_pkey` itself in case there was some padding in the struct.